clang/include/clang/Driver/Options.td

```diff
@@ -3162,6 +3162,12 @@ def modules_reduced_bmi : Flag<["-"], "fexperimental-modules-reduced-bmi">,
   HelpText<"Generate the reduced BMI">,
   MarshallingInfoFlag<FrontendOpts<"GenReducedBMI">>;
 
+def fmodules_embed_all_files : Joined<["-"], "fmodules-embed-all-files">,
```

Comment:
I moved the position since the original position is enclosed by a group assigning the CC1Option.